docs/src/main/asciidoc/security-authentication-mechanisms.adoc

```diff
@@ -779,6 +780,54 @@ quarkus.http.auth.permission.roles1.methods=GET <2>
 <2> Make the `roles1` permission match only the endpoint annotated with the `@AuthorizationCodeFlow` annotation.
 Unannotated endpoints must avoid the delay caused by the `applies-to=JAXRS` option.
 
+=== Use inclusive authentication to enable path-based authentication
+
+By default, Quarkus only supports <<http-security-policy-path-based-auth,path-based authentication>> for one authentication mechanism per path.
+If more than one authentication mechanism must be used for the path-based authentication, you can write a custom `HttpAuthenticationMechanism` as documented in the xref:security-customization.adoc#dealing-with-more-than-one-http-auth-mechanisms[Dealing with more than one HttpAuthenticationMechanism] section of the Security Tips and Tricks guide.
+Another option is to enable inclusive authentication in the lax mode and write a custom `HttpSecurityPolicy` or `PermissionChecker` that verifies that all registered HTTP authentication mechanisms created their mechanism-specific `SecurityIdentity`.
```

Comment:
```suggestion
Another option is to enable the <<inclusive authentication-link>> in the lax mode and write a custom `HttpSecurityPolicy` or `PermissionChecker` that verifies that all registered HTTP authentication mechanisms created their mechanism-specific `SecurityIdentity`.
```

Proposing to link here to the inclusive authentication subsection once it is created as proposed above